.pre-commit-config.yaml

```
@@ -66,7 +66,7 @@ repos:
           - "B101,B311"
 
   - repo: https://github.com/PyCQA/docformatter
-    rev: v1.6.0.rc1
+    rev: v1.5.1
     hooks:
       - id: docformatter
         args: [--in-place, --wrap-summaries=115, --wrap-descriptions=120]
```

```suggestion
    rev: v1.6.0.rc1
    hooks:
      - id: docformatter
        args: [--in-place, --wrap-summaries=115, --wrap-descriptions=120,  --pre-summary-space]
```

Let's try this -- from https://github.com/PyCQA/docformatter/issues/94#issuecomment-1368120910